### PR TITLE
CTP-2241: added Behat test

### PR DESCRIPTION
**Pull request by opitz**
checking that the alerts block will show the intended alerts.

### CODE REVIEW

**Review by aspark21 — COMMENTED** [review]
Should these alert URLs also be tested to work?

Are they meant to be clickable?

**Comment by opitz** [comment]
@aspark21 clicking them now...

**Comment by stuartlamour** [comment]
be aware the carousel aspect will probably be removed (it has never been used, and it not accessible) https://ucldata.atlassian.net/browse/CTP-3220 , so you can just test a single alert.

**Comment by opitz** [comment]
@stuartlamour no more carousel in tests now.

**Comment by stuartlamour** [comment]
> @stuartlamour no more carousel in tests now.

@opitz - apologies, by removing the carousel of alerts i mean the block will be updated so only a single alert can be added or shown at a time.

File: tests/behat/show_alerts.feature [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    Given the following "blocks" exist:
+      | blockname   | contextlevel | reference  | pagetypepattern | defaultregion |
+      | alerts      | System       | 1          | site-index      | side-pre      |
+    And the following config values are set as admin:
+      | config                  | value                           | plugin        |
+      | title1                  | Alert Title One                 | block_alerts  |
+      | description1            | Alert Description One           | block_alerts  |
+      | link1                   | https://nasa.gov                | block_alerts  |
+      | linktext1               | Alert Link One                  | block_alerts  |
+      | title2                  | Alert Title Two                 | block_alerts  |
```
Comment (stuartlamour) [inline tests/behat/show_alerts.feature]: Since there will only be one alert, you can remove the others.

Comment (opitz) [reply tests/behat/show_alerts.feature]: done.

**Comment by opitz** [comment]
all done now

**Review by stuartlamour — CHANGES_REQUESTED** [review]
pr does not include jira ticket number.

**Comment by opitz** [comment]
@stuartlamour added jira ticket number . please re-review.

**Review by stuartlamour — APPROVED** [review]

**Pull request merged**